doc/lua-filters.md

```diff
@@ -371,6 +371,56 @@ function Meta(m)
 end
 ```
 
+## Adding additional text
+
+This filter adds 'Hello World!' to the beginning of the document, and
+'Good Bye!' to the end of the document.
+
+
+```lua
+function Doc(doc)
+  table.insert(doc.blocks, 1, pandoc.Para{ pandoc.Str 'Hello World!' })
+  table.insert(doc.blocks, pandoc.Para{ pandoc.Str 'Good Bye!' })
+  return pandoc.Pandoc(doc.blocks, doc.meta)
+end
+```
+
+## Adding additional text to only specific output formats
+
+This filter adds 'Hello World!' to the beginning of the document, and
+'Good Bye!' to the end of the document, _only if_ the output format is
+`HTML`.
+
+```lua
+function Doc(doc)
```

Comment:
This doesn't quite work as described above, as there are other formats which can include raw HTML. Markdown and Org are examples. Using `if FORMAT:match 'html' then … end` would be closer.